neo/Network/P2P/ChannelsStartConfig.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Net;
+
+namespace Neo.Network.P2P
+{
+    public class ChannelsStartConfig
```

Comment:
Each object instantiated from this class is a single channel, right?
Maybe it should be `ChannelStartConfig`.
In addition, why to have this Start? Does it change during node operation in the channel?

Reply:
It configure more than one channel, websocket and Tcp

Reply:
I see, but those channels are fixed after nodes handshaked, yes?
Maybe it could be `ChannelsConfig` or `P2PChannelsConfig`? I do not know.
Maybe just leave as it is...ahheuaheuaea

Reply:
ChannelsConfig could be good